generators/form/templates/formSimple.js.ejs

```diff
@@ -0,0 +1,50 @@
+<%- include('partials/formHeader', {formNumber: formNumber}); %>
+
+const formFields = {
+  firstName: 'firstName'
+};
+
+const formConfig = {
+  urlPrefix: '/',
+  submitUrl: '/v0/api',
+  trackingPrefix: '<%= trackingPrefix %>',
+  introduction: IntroductionPage,
+  confirmation: ConfirmationPage,
+  formId: '<%= formNumber %>',
+  version: 0,
+  prefillEnabled: true,
+  savedFormMessages: {
+    notFound: 'Please start over to apply for <%= benefitDescription %>.',
+    noAuth: 'Please sign in again to continue your application for <%= benefitDescription %>.'
+  },
+  title: '<%= appName %>',
+  defaultDefinitions: {
+  },
+  chapters: {
+    chapter1: {
+      title: 'Personal Information',
+      pages: {
+        page1: {
+          path: 'first-name',
+          title: 'Personal Info - Page 1',
+          uiSchema: {
+            [formFields.firstName]: {
```

Comment:
I also prefer seeing the array access notation here; makes the form field properties stand out more, IMO.